logging/logging.go

```diff
@@ -298,13 +298,75 @@ func detectGAEResource() *mrpb.MonitoredResource {
 	}
 }
 
+func isCloudRun() bool {
+	_, config := os.LookupEnv("K_CONFIGURATION")
```

Comment:
Note that `K_CONFIGURATION` is part of the [knative](https://github.com/knative/serving/issues/2102) contract (which Cloud Run is based on), not just cloud run itself. This may misfire on other knative environments

This is something I'm not accounting for in Python currently.

Reply:
hmm good point, at the least I should look for other cloud run envvars, not just k_config

Reply:
I think the other k_* variables will be the same. It might be a good idea to check the metadata server too?

But this probably doesn't have to be addressed right now